roles/gitea/README.rst

```diff
@@ -0,0 +1,10 @@
+=============
+Gitea README
+=============
+
+This Ansible role installs Gitea.
```

Comment:
Maybe make this slightly more descriptive: eg.

This Ansible roles installs Gitea - a self-hosted Git service written in Go.